GitUI/CommitInfo/CommitInfo.cs

```diff
@@ -34,11 +34,7 @@ public partial class CommitInfo : GitModuleControl
         private readonly ICommitDataManager _commitDataManager;
         private readonly ICommitDataHeaderRenderer _commitDataHeaderRenderer;
         private readonly ICommitDataBodyRenderer _commitDataBodyRenderer;
-        private readonly IExternalLinksLoader _externalLinksLoader;
-        private readonly IConfiguredLinkDefinitionsProvider _effectiveLinkDefinitionsProvider;
         private readonly IGitRevisionExternalLinksParser _gitRevisionExternalLinksParser;
-        private readonly IExternalLinkRevisionParser _externalLinkRevisionParser;
-        private readonly IGitRemoteManager _gitRemoteManager;
```

Comment:
I would prefer to keeping these as they are. I have not specifically simplified/inlined these definitions as they make it easier to see what/how many external references each component has. They also make it easier to reason about future refactors.
Same applies to other instances.

Reply:
Did you want to revert all of 32a3fd7?

Reply:
No, only to https://github.com/gitextensions/gitextensions/pull/4621/commits/32a3fd780a8073a1322e5aa12d82835ceca20f57#diff-2dc8f6be53ee0e7dc3f0c7ca72b04d73, if possible
The rest looks fine

Reply:
np. I undid changes to `CommitInfo.cs`. Rebased on master and pushed.